src/framework/MOM_domains.F90

```diff
@@ -37,7 +38,7 @@ module MOM_domains
 
 public :: MOM_domains_init, MOM_infra_init, MOM_infra_end, get_domain_extent, get_domain_extent_dsamp2
 public :: MOM_define_domain, MOM_define_io_domain, clone_MOM_domain
-public :: pass_var, pass_vector, PE_here, root_PE, num_PEs
+public :: pass_var, pass_vector, PE_here, root_PE, num_PEs, Get_PElist
```

Comment:
I don't think making Get_PElist public will be acceptable. If you need the PElist, you can use follow what is done in the mom_cap at [L860](https://github.com/NOAA-EMC/MOM6/blob/d531a324a91f5de652055cbd8adfd84c4e75be30/config_src/nuopc_driver/mom_cap.F90#L860)

Reply:
OK, i will follow mom_cap.